sdk/appconfiguration/app-configuration/test/samplesForApiReview.ts

```diff
@@ -0,0 +1,107 @@
+import { AppConfigurationClient, isFeatureFlag, isSecretReference } from "../src";
+import { isFeatureFlagClientFilter } from "../src/featureFlag";
+
+const appConfigClient = new AppConfigurationClient("connection-string");
+
```

Comment:
For feature flag and secret references
- Add samples 
- Add snippets in readme

Reply:
So if you want to demo keyvault in the sample will that make keyvault a prereq in the rush.json build graph? I think it's fine, but it'll be interesting. :)

Reply:
Hmm.. I'll see.

Reply:
I'm depending on beta.. I don't see a reason not to.
I am assuming the customers are smart enough to understand.